src/styles/blocks/mainn.scss

```diff
@@ -0,0 +1,3 @@
+.main__products {
```

Comment:
I think the name of this file (mainn.scss) might be confusing. Also, strictly speaking, there isn't a block called "mainn" in your code. I understand why you named it like this, and I had the same problem, but IMHO mentors won't like it.

Reply:
named it as main-content.